src/components/Projects/Projects.jsx

```diff
@@ -121,6 +123,44 @@ const Projects = function(props) {
     await props.postNewProject(name, category);
   };
 
+  // Fetch autocomplete suggestions
+  useEffect(() => {
+    const fetchSuggestions = async () => {
+      if (debouncedSearchName) {
+        const userSuggestions = await props.getUserByAutocomplete(debouncedSearchName);
+        if (userSuggestions) {
+          setSuggestions(userSuggestions);
```

Comment:
I recommend wrapping the fetchSuggestions call in a try-catch block to gracefully handle any errors and provide feedback to the user.

Reply:
Fixed it @navya9989.